README.md

```diff
@@ -40,7 +40,15 @@ giving it a unique and appealing aesthetic in line with Catppuccin's color palet
 
 ## 🖥️ Install
 
-To install these userstyles:
+To install all userstyles with the recommended settings:
+
+1. Download [the Stylus export](./compiled.json?raw=true).
+2. Open the Stylus "manage" page.
+3. Locate backup (on the left bar) and select import.
+   - Select the downloaded file.
+4. Enjoy!
+
+To install individual userstyles:
 
 1. Install [Stylus](https://github.com/openstyles/stylus).
```

Comment:
I'd make use of subheadings (`##`) to give the steps some visual prominence, also it's a bit odd how the steps assume different starting points, the everything section assumes you already have stylus installed but the individual one doesn't? You should move up the stylus install instructions to the top level, something like the following:

```
## Install

<stylus install instructions>

### Everything
<how to import the compiled json>

### Individual
<how to install the individual theme>
```